contrib/cirrus/lib.sh

```diff
@@ -61,22 +61,17 @@ CONTINUOUS_INTEGRATION="${CONTINUOUS_INTEGRATION:-false}"
 CIRRUS_REPO_NAME=${CIRRUS_REPO_NAME:-libpod}
 CIRRUS_BASE_SHA=${CIRRUS_BASE_SHA:-unknown$(date +%s)}  # difficult to reliably discover
 CIRRUS_BUILD_ID=${CIRRUS_BUILD_ID:-$RANDOM$(date +%s)}  # must be short and unique
+
+OS_RELEASE_ID="$(source /etc/os-release; echo $ID)"
+# GCE image-name compatible string representation of distribution _major_ version
+OS_RELEASE_VER="$(source /etc/os-release; echo $VERSION_ID | cut -d '.' -f 1)"
+# Combined to ease soe usage
+OS_REL_VER="${OS_RELEASE_ID}-${OS_RELEASE_VER}"
+
 # Vars. for image-building
-PACKER_VER="1.4.2"
-# CSV of cache-image names to build (see $PACKER_BASE/libpod_images.json)
-
-# List of cache imaes to build for 'CI:IMG' mode via build_vm_images.sh
-# Exists to support manual single-image building in case of emergency
-export PACKER_BUILDS="${PACKER_BUILDS:-ubuntu-20,ubuntu-19,fedora-32,fedora-31}"
-# Google cloud provides these, we just make copies (see $SCRIPT_BASE/README.md) for use
-export UBUNTU_BASE_IMAGE="ubuntu-2004-focal-v20200506"
-export PRIOR_UBUNTU_BASE_IMAGE="ubuntu-1910-eoan-v20200211"
-# Manually produced base-image names (see $SCRIPT_BASE/README.md)
-export FEDORA_BASE_IMAGE="fedora-cloud-base-32-1-6-1588257430"
-export PRIOR_FEDORA_BASE_IMAGE="fedora-cloud-base-31-1-9-1588257430"
-export BUILT_IMAGE_SUFFIX="${BUILT_IMAGE_SUFFIX:--$CIRRUS_REPO_NAME-${CIRRUS_BUILD_ID}}"
+export BUILT_IMAGE_SUFFIX="${_BUILT_IMAGE_SUFFIX:-$CIRRUS_REPO_NAME-${CIRRUS_BUILD_ID}}"
```

Comment:
Is this default (`reponame-buildid`) still valid nowadays when the convention is 'c followed by a bignum' ?

Reply:
This is some legacy garbage from when `hack/get_ci_vm.sh` didn't know how to interpret `.cirrus.yml` (now it does).  I wonder if maybe I should try getting rid of this line + `s/BUILD_IMAGE_SUFFIX/_BUILD_IMAGE_SUFFIX/` everywhere.

It'll take running the tests again, but maybe removing this legacy garbage (finally) is worth.  I'd also more seriously consider killing off the commented section Dan was concerned about (above).  I'm thinking it's more distraction than help, even if I need to re-enable it.